tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/jms/client/SelectorTranslatorTest.java

```diff
@@ -204,6 +204,37 @@ public void testParseJMSType() {
       checkNoSubstitute("JMSType");
    }
 
+   @Test
```

Comment:
The proper test would been on the compatibility-tests


I have added it as part of my merge. just letting you know for future reference.